wger/urls.py

```diff
@@ -222,6 +222,9 @@
         nutrition_api_views.search,
         name='ingredient-search'),
     url(r'^api/v2/', include(router.urls)),
+    url(r'^api/v2/exercise/(?P<id>\d+)/$',
```

Comment:
Any changes on this @nyandiekaFelix ?

Reply:
I changed the parameter `name='exerciseinfo'` to `name='exercise'`. Anything else I need to change?